### ISSUE TITLE
Implement Transaction Status page (Sending / Confirmed / Finalised)

### ISSUE THREAD

**Issue by briansztamfater** [status-im/status-react#16934]
# Feature Issue

### User Story

As a user, I want to see a the state of my transaction in real time **so that** I can be sure if it the transaction succeeds or not

### Description

Figma Design 1: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1082-261340&mode=design&t=aTiN1IeemgbJnFPl-4
Figma Design 2: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1084-261948&mode=design&t=aTiN1IeemgbJnFPl-4
Figma Design 3: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1091-266847&mode=design&t=aTiN1IeemgbJnFPl-4
Figma Design 4: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1091-271993&mode=design&t=aTiN1IeemgbJnFPl-4
Figma Design 5: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1091-272706&mode=design&t=aTiN1IeemgbJnFPl-4

Visual reference:
<img width="150" src="https://github.com/status-im/status-mobile/assets/18485527/35ccf95c-3b39-49d1-a162-c7c87f077c25"> <img width="150" src="https://github.com/status-im/status-mobile/assets/18485527/3c82d89e-a69c-4213-b93f-92c2f9b12112"> <img width="150" src="https://github.com/status-im/status-mobile/assets/18485527/8525a598-c2f2-485a-a33e-86fe8cf5f298"> <img width="150" src="https://github.com/status-im/status-mobile/assets/18485527/9e440fb2-98fe-4f90-8efa-7f2add3462cb"> <img width="150" src="https://github.com/status-im/status-mobile/assets/18485527/f98f152c-3c41-4cf0-bc78-06dccfe024d0">

### Acceptance Criteria

- [ ] Show each operation status for each network (`Sending` / `Bridging and sending`) 
- [ ] If there are transactions pending, show `Sending...` title as in Figma designs 1 and 2. If transactions are confirmed, show `Transactions confirmed!` as in Figma designs 3 and 4. If transactions are finalised on each network (reaches minimum amount of blocks), show `Transaction finalised!` as Figma design 5
- [ ] User can dismiss the screen at any moment by tapping the `Done` button and reopen by tapping in the transaction item within the `Activity` tab in wallet. If `Activity` tab in wallet is not created yet, the reopen part of this item can be implemented in another PR.
- [ ] Background gradient should represent the account color
- [ ] Light and dark mode should be implemented

Feel free to break up this issue in smaller issues if necessary

**Comment by J-Son89** [comment]
blocked by - https://github.com/status-im/status-mobile/issues/18307

**Comment by J-Son89** [comment]
let's hold off on this issue for the time being as there is more priority issues to deliver 👍
